docs/dictionary/control_st/next-repeat.lcdoc

```diff
@@ -25,10 +27,9 @@ example performs the loop action only if the <current card|current
 card's> name contains the letter "e" :
 
     repeat for the number of cards of this stack
```

Comment:
@livecodeali @livecodesam what do you think about this example? Perhaps something different as there's definitely better ways to build a list of card names containing the letter `e`:

```
get the cardNames of this stack
filter lines of it with "*e*"
```

Reply:
Also I suspect it would be a good policy for our examples not to use single line `if`

Reply:
How about something common:
```
local tFolders
repeat for each line tFolder in folders(tPath)
    if tFolder begins with "." then
        next repeat
    end if
    put tPath & slash & tFolder & return after tFolders
end repeat
```